tools/run_tests/sanity/check_submodules.sh

```diff
@@ -27,7 +27,7 @@ want_submodules=$(mktemp /tmp/submXXXXXX)
 git submodule | awk '{ print $1 }' | sort > "$submodules"
 cat << EOF | awk '{ print $1 }' | sort > "$want_submodules"
  5b7683f49e1e9223cf9927b24f6fd3d6bd82e3f8 third_party/benchmark (v1.2.0)
- a20bb7ff8bb5057065a2e7941249773f9676cf45 third_party/boringssl (heads/chromium-stable)
+ 70ef9596bbcc11353b9bb8d4e91478694dd21439 third_party/boringssl (fips-20170615-704-g70ef9596)
```

Comment:
Is `fips-20170615-704-g70ef9596` the name of the branch we're using?

Reply:
May be the coincide?

Reply:
yes, they coincide I think. I definitely updated to chromium-stable.  There are rules by which git determines which one is the "primary" name of a given revision, but those priorities aren't set by most projects I've seen AFAIK.

Reply:
currently chromium-stable is at  70ef9596
https://github.com/google/boringssl/commits/chromium-stable